docs/src/CommutativeAlgebra/groebner_bases.md

```diff
@@ -211,38 +211,44 @@ Then the *leading ideal* of $G$ is the ideal of $K[x]$ defined by
 
 $\text{L}_>(G)=\langle \text{LT}_>(g) \mid g\in G\setminus\{0\}\rangle\subset K[x].$
 
-A *standard basis* of an ideal $I\subset K[x]_>$ is a finite subset $G$ of $I$ such that $\text{L}_>(G) = \text{L}_>(I)$.
-A finite subset of $K[x]_>$ is a *standard basis* if it is a standard basis for the ideal it generates.
+A finite subset $G$ of an ideal $I\subset K[x]_>$ is called a *standard basis* of $I$ (with respect to $>$)
+if $\text{L}_>(G) = \text{L}_>(I)$.  A finite subset of $K[x]_>$ is a *standard basis*
+if it is a standard basis of the ideal it generates. A standard basis with respect to a global monomial
+ordering is also called a *Gröbner basis*.
 
-We call a standard basis $G = \{g_1,\dots, g_r\}\subset K[x]_>\setminus \{0\}$ *minimal*  if  $\text{LM}_>(g_i)\neq \text{LM}_>(g_j)$
-for $i\neq j$.
+!!! note
+    Every standard basis of $I$ generates $I$.
 
 !!! note
-    The definition above deviates from the definition in most textbooks as we do not ask that the leading coefficients of the standard basis elements are 1.
+    Gröbner bases (standard bases) can be computed using Buchberger's algorithm (Buchberger's as algorithm enhanced by Mora).
```

Comment:
typo ? Additional 'as' in bracket

Reply:
@ederc  Can you please change this to:   Buchberger's algorithm as enhanced by Mora